xarray/coding/strings.py

```diff
@@ -221,7 +222,7 @@ class StackedBytesArray(indexing.ExplicitlyIndexedNDArrayMixin):
     values, when accessed, are automatically stacked along the last dimension.
 
     >>> indexer = indexing.BasicIndexer((slice(None),))
-    >>> StackedBytesArray(np.array(["a", "b", "c"], dtype="S1"))[indexer]
+    >>> StackedBytesArray(np.array(["a", "b", "c"], dtype="S1"))[indexer.tuple]
```

Comment:
```suggestion
    >>> StackedBytesArray(np.array(["a", "b", "c"], dtype="S1"))[slice(None)]
```

let's also delete the previous line. These docstrings won't have access to `*Indexer`